src/server/scripts/EasternKingdoms/ZulAman/boss_halazzi.cpp

```diff
@@ -123,27 +125,25 @@ struct boss_halazzi : public BossAI
     void DamageTaken(Unit* /*attacker*/, uint32& damage, DamageEffectType /*damagetype*/, SpellSchoolMask /*damageSchoolMask*/) override
```

Comment:
We are overriding DamageTaken, so calling the parent class function (BossAI::DamageTaken()) is required, else SetInvincibility() will have no effect